repos/fdbt-site/src/interfaces/index.ts

```diff
@@ -734,3 +734,20 @@ export interface Cap {
     durationAmount: string;
     durationUnits: ExpiryUnit;
 }
+
+export interface MultiTapPricingAttribute {
```

Comment:
I dont think we need to add the interfaces here. Instead, in the sessions page we can set the contents of the attribute equal to something. I will comment in sessions to show you what I mean.

Reply:
These are still here, like I said I dont think we need them. MultiTapPricing is fine, but the tapPriceId doesnt seem useful.